tests/fixtures/ead/ead_record_blank_optional_fields.xml

```diff
@@ -11,10 +11,22 @@
                 <archdesc level="collection">
                     <did>
                         <unittitle></unittitle>
+                        <unitdate></unitdate>
+                        <unitdatestructured>
```

Comment:
We should remove `unitdatestructured` from this fixture too since we aren't using it.